readme.md

```diff
@@ -13,20 +13,24 @@ the square brackets (for multiple-choice questions).
 
 ### Question #1
 
-Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
+Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back.
+(Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
 ```js
-
+jQuery.ajax( "http://kittengifs.com/gifs" [results] )
+console.log(results)
 ```
 
 ### Question #2
 
-Describe at a high level how we use jQuery to submit a form via AJAX.
+Describe at a high level how we use jQuery to submit a form via
+AJAX.
 
 Your Answer:
 ```text
-
+jQuery makes a request to AJAX and uses the routers and controllers
+to push the form to AJAX.
 ```
```

Comment:
First we need to (1) create an event listener that triggers the AJAX request and (2) pull that information from the form using some sort of selector (e.g., `$( "input" ).val( );`).

The rest is making sure we submit the AJAX call properly. To clarify, we are not making a request TO ajax. We are using AJAX to make a call to some API. AJAX is the method - not the source of the information we're requesting.